include/net/cloud.h

```diff
@@ -53,6 +53,7 @@ enum cloud_endpoint {
 	CLOUD_EP_TOPIC_STATE,
 	CLOUD_EP_TOPIC_CONFIG,
 	CLOUD_EP_TOPIC_PAIR,
+	CLOUD_EP_TOPIC_BATCH,
```

Comment:
I assume this is the topic that's intended for batch updates of some sorts? I'm not quite sure about the naming, but don't have any better suggestions either..

Reply:
Yes, you are correct. It is currently used in the [cat tracker](https://github.com/bifravst/cat-tracker-fw) to publish buffered gps fixes in batches to a seperate topic, as per request from the frontend developer. I guess it makes it easier to seperate "historical" data from the latest sensor sample, which is typically published with CLOUD_EP_TOPIC_MSG.

Reply:
Thanks for explaining.

Reply:
All right, thanks.